envs/ucsc.yaml

```diff
@@ -7,3 +7,4 @@ dependencies:
   - bioconda::ucsc-bedsort=377
   - bioconda::ucsc-bedGraphToBigWig=377
   - bioconda::ucsc-bedToBigBed=377
+  - bioconda::ucsc-fatotwobit=377
```

Comment:
ucsc.yaml is for the track-/assembly hub? To me, the purpose is unclear from the name.

Reply:
All the ucsc related tools are there, not the dependencies for the trackhub/assembly hub. Feel free to change it to sth more clear